src/style.css

```diff
@@ -1,3 +1,52 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.logo {
+  margin-left: 50px;
+  padding-top: 10px;
+}
+
+.main_nav-link {
+  display: flex;
+  gap: 20px;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font-family in case the main one doesn't work. For example: `font-family: Roboto, Arial, sans-serif;` could be `font-family: Roboto, 'Arial', sans-serif;` to ensure there's a default font that's widely available.